requirements/requirements.in

```diff
@@ -65,3 +65,7 @@ boto3
 
 # Matplotlib
 mpld3
+
+# External Resource Plugins
+## Ray
+ray>=2.2.0
```

Comment:
Recall that adding something to `requirements.in` only adds it to our bazel managed python envs, NOT to the wheel we ship to customers. It will only be available to bazel targets/tests that have `ray` as a dependency (directly or transitively).

Reply:
That means we should have no targets adding Ray as a dependency, right?